src/components/UserProfile/PersonalDetailsForm.js

```diff
@@ -26,6 +26,11 @@ export default function PersonalDetailsForm(props) {
         setGender(event.target.value);
     };
 
+    const [isActive, setIsActive] = useState(props?.user?.is_active || false);
```

Comment:
this code isn't merged. Did you work on the same branch? Always branch off of development

Reply:
I think maybe it is from the branch I made at first for setting IsActive. I think that needs to be merged first before this one.

Reply:
you branched off of your other feature branch. You should create a new branch off of master and move the changes relevant to this work over

Reply:
I made a new branch and pushed it. Is this one good?